### PR TITLE
Expose and manage system media types

### PR DESCRIPTION
**Pull request by kjac**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

### Description

The core is depending on having certain media types available - the "system" media types (File, Folder, Image). Whilst not ideal, this is the current state of the core.

We need to ensure that these are not deleted or have their alias changed.

The V13 approach was purely client-side. This is enabled by this PR for V14 by adding `IsDeletable`  and `AliasCanBeChanged` properties to the relevant media type endpoints.

Additionally, this PR ensures that we do not accept critical changes to the system media types at service level.

### Testing this PR

The media type tree endpoints should list the system media types as non-deletable (`IsDeletable == false`).

The "by ID" endpoint should also list the system media types as non-deletable, and also non-alias-changeable 😆 (`AliasCanBeChanged == false`).

### CODE REVIEW

File: src/Umbraco.Web.UI/umbraco/Licenses/umbracoForms.lic
Comment (Migaroez) [inline src/Umbraco.Web.UI/umbraco/Licenses/umbracoForms.lic:1]: woopsie?

**Review by Migaroez — CHANGES_REQUESTED** [review]
Works as described, code looks ok ✅ 
Todo: cleanup license file commit
Optional: add a unittest for the delete behaviour?

File: src/Umbraco.Web.UI/umbraco/Licenses/umbracoForms.lic
Comment (kjac) [reply src/Umbraco.Web.UI/umbraco/Licenses/umbracoForms.lic:1]: Geez. That explains why Git was complaining so much ... thanks 🤦 it's been removed.

**Comment by kjac** [comment]
There is a test for the delete behavior 😄 https://github.com/umbraco/Umbraco-CMS/pull/16343/files#diff-0603ca1f3e26b7b478be4d08bca0fae507da4ddadbb0f1f1c00e6e7859d37160R226-R235

**Review by Migaroez — APPROVED** [review]
After being blind on both ends, seems to be in a good state now 😄

**Pull request merged**